Enterprise-CMCS/macpro-mdct-seds
# CMDCT-3224: LoadData begone

### Description
This PR removes the "Load Data" page from SEDS.

This page was probably never intended to be seen by users. It has no place in our frontend.

### Related ticket(s)
CMDCT-3224

---
### How to test
* Log into the site, as an admin
* Edit the URL to `...#/load-data`
* Note that you do _not_ see a page with a file upload input

### Important updates
n/a

---
### Author checklist
